File: pset6/caesar.py
import sys
from sys import argv
from cs50 import get_string

encrypted = ""
syntax = ""

if len(sys.argv) != 2:
    print("missing command-line argument")
    sys.exit(1)

key = int(sys.argv[1])
plaintext = get_string("plaintext: ")
print("ciphertext: ", end="");
for letter in plaintext:

    if (letter.islower()):
          print(chr((ord(letter)-ord('a')+key)%26 +ord('a')), end="")
    elif (letter.isupper()):
         print(chr((ord(letter)-ord('A')+key)%26 +ord('A')), end="")
    else:
         print(letter, end="")
print()

File: pset2/caesar.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <ctype.h>



int main(int argc, string argv[])
{

if (argc != 2)
  {
      printf("Usage key %s\n",argv[0]);
      return 1;
  }
else
  {

    for (int i = 0, n = strlen(argv[1]); i < n; i++)
    {
          if ( isalpha (argv[1] [i]))
          {
              printf("Usage key %s\n",argv[0]);
              return 1;
          }

    }
      int key = atoi(argv[1]);
      if ( key > 0)
      {
          string s = get_string("plaintext: ");
          printf("ciphertext: ");
          for (int j = 0; j < strlen(s); j++)
              {
              if(islower(s[j]))
                 {
                     printf("%c",(((s[j]+key)-97)%26)+97);
                 }

              else if (isupper(s[j]))
                 {
                     printf("%c",(((s[j]+key)-65)%26)+65);
                 }
              else
                 {
                     printf("%c", s[j]);
                 }
              }

          printf("\n");
      }
 }
}